install_config/aggregate_logging.adoc

```diff
@@ -34,6 +34,10 @@ all logs.
 
 This topic shows you how to deploy the stack.
 
+link:../install/prerequisites.html#managing-docker-container-logs[Managing
+Docker Container Logs] discusses the use of `json-file` logging driver options
+to manage your container's log when it becomes full.
```

Comment:
I think this could be in a "Note" admonition since it's a bit of a tangent (though related) and takes you to a different topic.